scio/pom.xml

```diff
@@ -43,6 +43,9 @@
     <plugin.shade.version>2.3</plugin.shade.version>
     <plugin.scala.version>2.15.2</plugin.scala.version>
     <plugin.avro.version>1.7.7</plugin.avro.version>
+
+    <!--test library versions-->
+    <hamcrest.all.version>1.3</hamcrest.all.version>
```

Comment:
see https://github.com/apache/zeppelin/pull/2759/files#diff-608f20f45ac75cfdb867d8d83eb0216bR98?

Reply:
this should be added as test scope

Reply:
**hamcrest-all** is already set as test scope (see line 107ff).